src/Solution/DirectorySolution.php

```diff
@@ -62,6 +80,17 @@ public static function fromDirectory($directory, $entryPoint = 'solution.php')
     {
         return new static($directory, $entryPoint);
     }
+
+    /**
+     * @param array  $exclusions
+     * @param string $directory
+     * @param string $entryPoint
+     * @return static
+     */
+    public static function fromDirectoryWithExclusions(array $exclusions, $directory, $entryPoint = 'solution.php')
```

Comment:
damn of course!! are was questioning myself when I created the method. 

Nice spot dude :+1: